odoo/addons/base/models/ir_cron.py

```diff
@@ -474,7 +474,7 @@ def _trigger(self, at=None):
             at_list = list(at)
             assert all(isinstance(at, datetime) for at in at_list)
 
-        self._trigger_list(at_list)
+        return self._trigger_list(at_list)
```

Comment:
you forgot to update the docstring with a `:return: the created triggers records` `:rtype: recordset`